RELEASE_PROCESS.md

```diff
@@ -22,7 +22,7 @@
     │       ├───Microsoft.Azure.WebJobs.Extensions.RabbitMQ.2.0.0-preview.symbols.nupkg
     │       └───_manifest
     │           └───<files>
-    └───drop-java-bindings
+    └───drop-java-library
         └───2.0.0-preview
             ├───_manifest
             │   └───<files>
```

Comment:
Modified three files: `RELEASE_PROCESS.md`, `azure-pipeline.yml`, and `.gitignore`. Rest of all files are only shifted to different folders.